lib/open_project/scm/adapters/git.rb

```diff
@@ -269,41 +243,51 @@ def revisions(path, identifier_from, identifier_to, options = {})
             end
           end
           revisions
-        rescue ScmCommandAborted
-          revisions
+        end
+
+        def build_revision_args(path, identifier_from, identifier_to, options)
```

Comment:
Assignment Branch Condition size for build_revision_args is too high. [24.78/15]